nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -93,6 +93,176 @@ let
       occ "$@"
   '';
 
+  dbBackupRestore =
+    if cfg.config.dbtype == "sqlite"
+    then
+      {
+        # The sqlite database is automatically backed up since it's contained in the datadir
+        backupFileName = "data/nextcloud.db";
```

Comment:
`nextcloud` above is actually `${config.services.nextcloud.config.dbname}`.

Reply:
I know about that option because I came to NixOS nextcloud module from an old owncloud install -- which required me to set `dbname = "owncloud"`.

Reply:
Thanks, I fixed it in the latest version.